docs/stylesheets/extra.css

```
@@ -1,3 +1,7 @@
 :root {
     --md-primary-fg-color:        #0000CD;
   }
+.md-content__inner img {
+  box-sizing: content-box;
+  width: 80%;
```

This is a little annoying because they're still all different sizes. But I don't think they look as bad at different sizes when they're figures like this.

Maybe use `max-width` so we don't stretch really small images?
```suggestion
  max-width: 80%;
```